src/components/App.js

```diff
@@ -1,14 +1,18 @@
-import React from 'react';
-import Display from './Display';
-import ButtonPanel from './ButtonPanel';
-
-const App = function app() {
-  return (
-    <>
-      <Display />
-      <ButtonPanel />
-    </>
-  );
-};
+/* eslint-disable no-unused-vars */
+import React, { useState, Component, Fragment } from 'react';
+// import ButtonPannel from './ButtonPanel';
+// import Display from './Display';
+// import Button from './Button';
+// import calculate from '../logic/calculate';
```

Comment:
- [ ] Please let's clean up these newly created commented codes from the codebase.